### PR TITLE
Add documentation for Feb Providers release

### PR DESCRIPTION
**Pull request by potiuk**
This provider's release is pretty special as we release
all providers with Python 3.10 support.

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/main/UPDATING.md).

### CODE REVIEW

**Comment by potiuk** [comment]
Finally after some dependency struggles I prepared release for providers. This is a special release - we are going to release ALL providers, because we are adding Python 3.10 support for all of them (so we need to update meta-data in the packages (technically we do not have, to, but this is much better if we explicitly mark all providers as Python 3.10 capable). 

Most of the providers are just those "Support Python 3.10".  

Once we release those in PyPI, we will be able to finally add full 3.10 suport for Airlfow in `main`

File: airflow/providers/datadog/CHANGELOG.rst [hunk 1]
```diff
@@ -19,6 +19,31 @@
 Changelog
 ---------
 
+2.0.2
+.....
+
+Features
+~~~~~~~~
+
+* ``Add documentation for January 2021 providers release (#21257)``
```
Comment (eladkal) [inline airflow/providers/datadog/CHANGELOG.rst]: is this one a mistake?
(I don't see this entry in other providers)

Comment (potiuk) [reply airflow/providers/datadog/CHANGELOG.rst]: Yep! Hawkeye :)

Comment (potiuk) [reply airflow/providers/datadog/CHANGELOG.rst]: Fixed!

**Review by eladkal — APPROVED** [review]

**Pull request merged**